### DISCUSSION TITLE
[Alerting] Add swit alert

### DISCUSSION THREAD

**Discussion by ben-swit** [grafana/grafana#41133] [Alerting]
Swit (https://swit.io) supports incoming webhook  

different between grafana and swit default webhook format 

how about add support swit alert 

PR implementing this feature request: #40916

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
